src/number_theory/padics/padic_norm.lean

```diff
@@ -407,15 +442,13 @@ end padic_val_nat
 
 section padic_val_nat
 
-/--
-If a prime doesn't appear in `n`, `padic_val_nat p n` is `0`.
--/
+/-- If a prime doesn't appear in `n`, `padic_val_nat p n` is `0`. -/
 lemma padic_val_nat_of_not_dvd {p : ℕ} [fact p.prime] {n : ℕ} (not_dvd : ¬(p ∣ n)) :
```

Comment:
Does this duplicate padic_val_nat.eq_zero_of_not_dvd (line 87)?

Reply:
Apparently yes. Not sure why this lemma had a p.prime hypothesis.